microcosm_pubsub/handlers/uri_handler.py

```diff
@@ -54,6 +56,12 @@ def nack_if_not_found(self):
     def resource_type(self):
         return dict
 
+    def get_resource_cache(self, graph):
+        try:
+            return graph.resource_cache
+        except (LockedGraphError, NotBoundError):
```

Comment:
I've seen NotBoundError used for similar check in `metrics.py`, however I got a `LockedGraphError`, so checking for both.